src/ml_tooling/metrics/permutation_importance.py

```diff
@@ -80,14 +99,18 @@ def permutation_importance(
     .. [BRE] L. Breiman, "Random Forests", Machine Learning, 45(1), 5-32,
              2001. https://doi.org/10.1023/A:1010933404324
     """
+    if hasattr(X, "iloc"):
+        X = X.copy()  # Dataframe
+    else:
+        X = check_array(X, force_all_finite="allow-nan", dtype=np.object, copy=True)
 
-    X = X.copy()
     random_state = check_random_state(random_state)
     scorer = check_scoring(estimator, scoring=scoring)
 
     baseline_score = scorer(estimator, X, y)
+    scores = np.zeros((X.shape[1], n_repeats))
 
-    scores = Parallel(n_jobs=n_jobs)(
+    scores = Parallel(n_jobs=n_jobs, max_nbytes=None)(
```

Comment:
Det er fair nok at vi forcer den igennem. Det er ikke noget problem for os tænker jeg.

Men jeg synes måske, at det skal beskrives et sted, at det er det vi gør. Evt i noget docstring. Her tænker jeg mere hvis eksterne skal bruge det til et eller andet.